### DISCUSSION TITLE
Recently Updated shows Hidden Posts

### DISCUSSION THREAD

**Discussion by andyhodgetts** [cotes2020/jekyll-theme-chirpy#1560] [Ideas]
Hi.  I noticed that the 'Recently Updated'  side panel includes posts that are marked as hidden.

A simple fix could be to utilise the filtering used on the Home page and modify '_includes/update-list.html', add the filter then enumerate over it:

from:
```
...
{% for post in site.posts %}
...
```

to:
```
...
{% assign not_hidden = site.posts | where_exp: 'item', 'item.hidden != true' %}

{% for post in not_hidden%}
...
```

**Comment by cotes2020** [comment]
Jekyll has better support for the variable `published`, so if you want to hide a post, I suggest you use `published: false` instead of `hidden: true`.

```yml
---
published: false
---
```

**Reply by kungfux** [reply]
In addition, let me remind you about [drafts](https://jekyllrb.com/docs/posts/#drafts) feature that may be useful too.

**Reply by ghost** [reply]
@kungfux @cotes2020 
1) are there ways to hide specific posts only on the homepage & recently updated section?
But it should show up in archive, category and tags.
2) Or just hide from homepage and show everywhere else?
Maybe by using some conditions like showing only pinned posts in home?
can I just remove the code in ``` <!-- Get default posts --> ``` section from ```_layouts/home.html```


> Edit: 2 worked 
Please let me know if 1 is possible

**Reply by kungfux** [reply]
> are there ways to hide specific posts only on the homepage & recently updated section?
But it should show up in archive, category and tags.

@B-and-B-Studio No, in general, rules are applied to every aspect, so if something should be hidden, it will be hidden everywhere, unless you manually adjust the logic by overriding each layout involved in displaying posts.

**Reply by ghost** [reply]
I see.
Thanks you 😊
